core/operation/src/main/java/uk/gov/gchq/gaffer/operation/OperationChain.java

```diff
@@ -97,23 +98,9 @@ void setOperationArray(final Operation[] operations) {
 
     @Override
     public String toString() {
-        final StringBuilder strBuilder = new StringBuilder("OperationChain[");
```

Comment:
We might need to check to see if some of these special implementations should be left as they are - there's a debate to be had around making the implementation uniform throughout the project vs keeping some flexibility to handle special cases like this. @p013570 might have an opinion on this.